src/App.jsx

```diff
@@ -1,104 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
```

Comment:
Ensure that the `PostList` component is styled according to the task description. The styles from `App.scss` should be moved to separate files, such as `PostList.scss`.